pkg/jvscrypto/kmsutil.go

```diff
@@ -180,21 +180,23 @@ func SignToken(token *jwt.Token, signer crypto.Signer) (string, error) {
 	return strings.Join([]string{signingString, jwt.EncodeSegment(sig)}, "."), nil
 }
 
-// getPrimary gets the key version name marked as primary in the key labels.
-func getPrimary(ctx context.Context, kms *kms.KeyManagementClient, key string) (string, error) {
+// GetPrimary gets the key version name marked as primary in the key labels.
+func GetPrimary(ctx context.Context, kms *kms.KeyManagementClient, key string) (string, error) {
 	response, err := kms.GetCryptoKey(ctx, &kmspb.GetCryptoKeyRequest{Name: key})
 	if err != nil {
 		return "", fmt.Errorf("issue while getting key from KMS: %w", err)
 	}
-	if primary, ok := response.Labels[primaryKey]; ok {
-		primary = strings.TrimPrefix(primary, valuePrefix)
+	if primary, ok := response.Labels[PrimaryKey]; ok {
+		primary = strings.TrimPrefix(primary, PrimaryLabelPrefix)
 		return fmt.Sprintf("%s/cryptoKeyVersions/%s", key, primary), nil
 	}
 	// no primary found
 	return "", nil
 }
 
 // SetPrimary sets the key version name as primary in the key labels.
+// 'Primary' field will be omitted for keys with purpose other than ENCRYPT_DECRYPT(https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.keyRings.cryptoKeys).
```

Comment:
nit: also mention what's the key type we're using.

Reply:
DONE